tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -100,6 +100,18 @@ public static function getContainer()
         return static::$kernel->getContainer();
     }
 
+    /**
+     * @AfterFeature @clear-downloads-after-feature
+     */
+    public static function clearDownloads(): void
```

Comment:
Woah this thing can be dangerous 😄 

Is `_PS_DOWNLOAD_DIR_` used for Behat tests different from regular `_PS_DOWNLOAD_DIR_` used for running the shop in localhost ? (for example is there a `test/download` for tests and `download` for prod)

Reply:
yes its the same. I could try redefining it in case its _PS_IN_TEST_ mode :thinking: